src/core.jl

```diff
@@ -73,11 +103,19 @@ The optional second argument take the `v`th vertex adjacency list, that is:
 fadj(g::SimpleGraph) = g.fadjlist
 fadj(g::SimpleGraph, v::Int) = g.fadjlist[v]
 
+_column(a::SparseMatrixCSC, i::Integer) = sub(a.rowval, a.colptr[i]:a.colptr[i+1]-1)
+
+fadj(g::SimpleSparseGraph, v::Int) = _column(g.m,v)
+fadj(g::SimpleSparseGraph) = @inbounds [fadj(g,i) for i in 1:nv(g)]
+
 """Returns the backwards adjacency list of a graph.
 For each vertex the Array of `dst` for each edge eminating from that vertex."""
 badj(g::SimpleGraph) = g.badjlist
 badj(g::SimpleGraph, v::Int) = g.badjlist[v]
 
+badj(g::SimpleSparseGraph, v::Int) = g.m[v,:]'.rowval
+badj(g::SimpleSparseGraph) = @inbounds [badj(g,i) for i in 1:nv(g)]
+
```

Comment:
we could store tril(m)' in order to get efficient access to badj(g,i) just like fadj